docs/src/roadmap.md

```diff
@@ -38,9 +38,9 @@ Some but not all of these tasks are summarized in the
     with JuMP's documentation.
 - Developer experience
   - Get JuMP’s unit tests running in less than two minutes. See [#1745](https://github.com/jump-dev/JuMP.jl/pull/1745).
-- All solvers should complete the transition to MOI.
+- All solvers should complete the transition to MOI. (**Done**)
```

Comment:
Excluding Pajarito/Pavito.

Reply:
And NLopt

Reply:
And Alpine.

So yay/nay do marking as done then?

Reply:
I'd say nay. It's not obviously done.

Reply:
I'd say nay too